doc/releases/schedule.rst

```diff
@@ -10,16 +10,24 @@ Current
 Timeline
 --------
 
-.. ceph_timeline:: releases.yml development mimic luminous kraken jewel infernalis hammer giant firefly emperor dumpling
+.. ceph_timeline:: releases.yml development nautilus mimic luminous kraken jewel infernalis hammer giant firefly emperor
```

Comment:
@hoerup why are you removing dumpling from http://docs.ceph.com/docs/master/releases/schedule/ ?

Reply:
Because  ceph_timeline function only accepts 12 parameters

Reply:
If you want to keep the dumpling column, then I need to expand https://github.com/ceph/ceph/blob/master/doc/_ext/ceph_releases.py

Reply:
@hoerup i see.. that does make sense. but i'd suggest defer to @noahdesu .

Reply:
Changing the parameter that allows us to render release information for more than 12 releases is OK. I think the question is should we keep displaying historical data past 12 releases? My thought is that we should keep supporting old releases until we need to render the table in a different format, then revist the quesiton. @neha-ojha thoughts?

Reply:
One argument against is that the 12 columns just manage to squeeze into the current layout, and an additional column might squeeze it too much ? 
http://docs.ceph.com/docs/master/releases/schedule/

Reply:
> Changing the parameter that allows us to render release information for more than 12 releases is OK. I think the question is should we keep displaying historical data past 12 releases? My thought is that we should keep supporting old releases until we need to render the table in a different format, then revist the quesiton. @neha-ojha thoughts?

If we can come up with a way to cleanly display more than 12 columns or add a "show more" like thing, then I think we should display historical data past 12 releases because technically those are part of Ceph timeline.

Reply:
as an alternative to "show more", would it make sense to split it into two different tables (or pages) one with the newer version and an archived releases ? This raises a new question though, how should the development track be rendered then ?

Reply:
> as an alternative to "show more", would it make sense to split it into two different tables (or pages) one with the newer version and an archived releases ? This raises a new question though, how should the development track be rendered then ?

Sure, I don't think I understand your second question completely

Reply:
> Sure, I don't think I understand your second question completely

Currently the "development" column shows dev releases from release 0.68 (Sept. 2013) and up til now. If splitting into current + archive - should the devel column then be split into 2 sections that corresponds with the 2 periods covered by each section (including an overlap)

Reply:
i think something like this would work:

```
[dev | octopus | nautilus | ...]
[... | bobtail | argonaut]
```

this might be as easy as repeating the CephReleases directive in the .rst file with different parameters.

Reply:
Idea: merge this PR as it is - just to make sure the docs are ajour, and afterwards maybe we can figure out a plan to show all the releases in a new PR ?

Reply:
If we go with that plan, then I think per @neha-ojha we should retain the historical data even if the table overflows horizontally.